user/pages/03.commerce2/02.developer-guide/09.promotions/02.create-a-condition/docs.md

```diff
@@ -4,4 +4,222 @@ taxonomy:
     category: docs
 ---
 
+Commerce conditions are used in when creating a Promotion or a Payment Gateway 
+for your store. 
+
+In this context, conditions are a set of plugins unique to the 
+commerce project. They are not connected in function or code to conditions in 
+core such as: Drupal\Core\Database\Query\Condition. 
+
+A promotion condition evaluated only when a coupon is applied to an order.
```

Comment:
That's not true. Conditions are always validated to find applicable promotions even if they have no coupon.